src/NexusMods.App.UI/Controls/Trees/FileTreeView.axaml

```diff
@@ -9,9 +9,6 @@
                                 xmlns:system="clr-namespace:System;assembly=System.Runtime"
                                 mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
                                 x:Class="NexusMods.App.UI.Controls.Trees.FileTreeView">
-    <Design.DataContext>
-        <trees:ModFileTreeDesignViewModel />
```

Comment:
Not strictly related to PR, but not sure why this was removed in the first place. Maybe during some quick rework? Figured I'd just make a quick note.

A basic copy of this is still in the Examples folder, at least.

Reply:
Most Design ViewModels were culled and murdered during the LoadoutItem transition period. The survivors were few and rare

Reply:
![](https://media1.tenor.com/m/9PTGVf4BLwYAAAAd/crying-emoji-dies.gif)